revscoring/datasources/site.py

```diff
@@ -0,0 +1,32 @@
+import mw
+
+from .datasource import Datasource
+
+
+def process_doc(session):
+    
+        doc = session.site_info.query(properties={'general', 'namespaces',
+                                                  'namespacealiases'})
+        return doc
+
+doc = Datasource("site.doc", process_doc, depends_on=['session'])
+
+
+def process_namespace_map(site_info_doc):
+    
+    aliases = site_info_doc.get('namespacealiases', [])
+    alias_map = {}
```

Comment:
Horray for tests pointing out old bugs! :S

On Mon, Mar 9, 2015 at 4:53 PM, Helder <notifications@github.com> wrote:

> In revscoring/datasources/site.py
> <https://github.com/halfak/Revision-Scoring/pull/60#discussion_r26074836>:
>
> > +from .datasource import Datasource
> > +
> > +
> > +def process_doc(session):
> > +
> > +        doc = session.site_info.query(properties={'general', 'namespaces',
> > +                                                  'namespacealiases'})
> > +        return doc
> > +
> > +doc = Datasource("site.doc", process_doc, depends_on=['session'])
> > +
> > +
> > +def process_namespace_map(site_info_doc):
> > +
> > +    aliases = site_info_doc.get('namespacealiases', [])
> > +    alias_map = {}
>
> This fixes 2d6b636#diff-288091a7883dc67d96ad359400b9ac0dR10
> <https://github.com/halfak/Revision-Scoring/commit/2d6b636b16b7760a5f904c3b83b835251d638b8b#diff-288091a7883dc67d96ad359400b9ac0dR10>
>
> Thanks! :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/halfak/Revision-Scoring/pull/60/files#r26074836>.
>